new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -244,4 +344,30 @@ private void restartOperatorAndVerify() {
     assertFalse(opPodNameNew.equals(opPodName),
         "The operator names before and after a restart should be different");
   }
+
+  private void checkServiceExistsAndPodReady() {
```

Comment:
In CommonTestUtils.java, we have a similar test method: 
public static void checkPodReadyAndServiceExists(String podName, String domainUid, String namespace)

Reply:
thanks for pointing it out ...

Reply:
done